x/feemarket/ante/suite/suite.go

```diff
@@ -240,8 +243,13 @@ func (s *TestSuite) RunTestCase(t *testing.T, tc TestCase, args TestCaseArgs) {
 		tc.StateUpdate(s)
 	}
 
-	if tc.RunPost && anteErr == nil {
-		newCtx, postErr = s.PostHandler(s.Ctx, tx, tc.Simulate, true)
+	if tc.RunPost {
+		newCtx, postErr = s.PostHandler(s.Ctx, tx, tc.Simulate, anteErr == nil)
```

Comment:
we should always run the post handler in these tests even if there is an ante handler error. https://docs.cosmos.network/v0.50/learn/advanced/baseapp#runmsgs